docs/product-assembly_pam-on-openshift/src/main/asciidoc/environment-authoring-ha-proc.adoc

```diff
@@ -0,0 +1,42 @@
+[id='environment-authoring-ha-proc']
+= Deploying a high-availability authoring environment
+
+To deploy a high-availability authoring environment, use the `rhpam-70-authoring-ha.yaml` template file. You can download this file from `{URL_BASE_GITHUB_PAM}/templates`.
+
+.Procedure
+. Use one of the following methods to deploy the template:
+.. In the OpenShift Web UI, select *Add to Project >  Import YAML / JSON* and then select or paste the `rhpam-70-authoring-ha.yaml` file. In the *Add Template* window, ensure *Process the template* is selected and click *Continue*.
+.. To use the OpenShift command line console, prepare the following command line:
++
+[subs="verbatim,macros"]
+----
+oc new-app -f <template-path>/rhpam-70-authoring-ha.yaml -p PARAMETER=value -p PARAMETER=value...
+----
++
+In this command line:
++
+* Replace `<project-name>` with the name of your project
+* Replace `<template-path>` with the path to the downloaded template file.
+* Use as many `ATTRIBUTE=value` pairs as needed to set the required attributes. You can view the template file to see descriptions for the attributes.
++
+. Set the following mandatory attributes:
+** *Application Name* (`APPLICATION_NAME`): the name of the OpenShift application. It is used in the default URLs for {CENTRAL} and {KIE_SERVER}. OpenShift also uses the application name to create a separate set of deployment configurations, services, routes, labels, and artifacts. You can deploy several applications using the same template into the same project, as long as you use different application names. 
+** *Business Central Server Certificate Name* (`BUSINESS_CENTRAL_HTTPS_NAME`): the name of the certificate in the keystore that you created in <<secrets-central-create-proc>>.
+** *Business Central Server Keystore Password* (`BUSINESS_CENTRAL_HTTPS_PASSWORD`): the password for the keystore that you created in <<secrets-central-create-proc>>.
+** *KIE Server Certificate Name* (`KIE_SERVER_HTTPS_NAME`): the name of the certificate in the keystore that you created in <<secrets-kie-create-proc>>.
+** *KIE Server Keystore Password* (`KIE_SERVER_HTTPS_PASSWORD`): the password for the keystore that you created in <<secrets-kie-create-proc>>.
+** *KIE Server ID* (`KIE_SERVER_ID`): the name of the server template on the {CENTRAL} that the {KIE_SERVER} is to join. Set this parameter to any value. If you do not set it and then the {KIE_SERVER} pod is restarted or scaled, each new instance of the server might join a new template. 
+** *ImageStream Namespace* (`IMAGE_STREAM_NAMESPACE`): the namespace where the image streams are available. If the image streams were already available in your OpenShift environment (see <<imagestreams-file-install-proc>>), the namespace is `openshift`. If you have installed the image streams file, the namespace is the name of the OpenShift project.
++
```

Comment:
Play with the + line breaks again here. The sections below are not flushing back left and indenting properly.

Reply:
I don't see a problem.

![image](https://user-images.githubusercontent.com/33361640/40236107-5b653d5a-5aa4-11e8-85bc-63639f386faa.png)

Reply:
As for indenting a bit further to the right that's intentional because I want it cleat they are all under step 2

Reply:
Not sure what's going on! Either I was crazy the day I reviewed or something has changed. All good.